check-procs/lib/check_procs.go

```diff
@@ -92,27 +95,26 @@ func run(args []string) *checkers.Checker {
 		}
 		cmdExcludePatRegexp = r
 	}
+	result := checkers.OK
+	var msg string
 	var resultrocStates []procState
```

Comment:
[nit] Declaration of `resultrocStates` would be better between line 101 and 102.